test/functional/apps/visualize/_area_chart.js

```diff
@@ -0,0 +1,216 @@
+define(function (require) {
+  var Common = require('../../../support/pages/Common');
+  var HeaderPage = require('../../../support/pages/HeaderPage');
+  var SettingsPage = require('../../../support/pages/settings_page');
+  var DiscoverPage = require('../../../support/pages/DiscoverPage');
+  var VisualizePage = require('../../../support/pages/VisualizePage');
```

Comment:
Not a blocker for this PR, but in the future take a look at the `requirefrom` module, which is already included in the project: https://www.npmjs.com/package/requirefrom

It doesn't solve the relative path stuff entirely, but it does mean we have fewer paths to update if we ever had to move files around.